### PR TITLE
using a letter versus name for sum and prod in Latex parsing

### PR DESCRIPTION
**Pull request by bhpayne**
Sometimes "Pi" and "Sigma" are used for prod and sum

#### References to other Issues or PRs
No issues reference this Latex 

#### Brief description of what is fixed or changed
Although "sum" and "prod" are the correct Latex symbols, sometimes "Pi" and "Sigma" are used in their place in Latex

#### Release Notes

* parsing
  * expanded "sum" and "prod" to "Sigma" and "Pi" in Latex

### CODE REVIEW

File: sympy/parsing/latex/_antlr/latexparser.py [hunk 1]
```diff
@@ -15,10 +15,11 @@
 from io import StringIO
 import sys
 
+
 def serializedATN():
     with StringIO() as buf:
         buf.write(u"\3\u608b\ua72a\u8133\ub9ed\u417c\u3be7\u7786\u5964\3")
-        buf.write(u"K\u01b2\4\2\t\2\4\3\t\3\4\4\t\4\4\5\t\5\4\6\t\6\4\7\t")
+        buf.write(u"M\u01b2\4\2\t\2\4\3\t\3\4\4\t\4\4\5\t\5\4\6\t\6\4\7\t")
```
Comment (eric-wieser) [inline sympy/parsing/latex/_antlr/latexparser.py:22]: Note: #19980 will make these changes collapsed by default in github.

**Comment by czgdp1807** [comment]
@sylee957 @eric-wieser  Is it good to merge?

File: sympy/parsing/latex/_antlr/latexparser.py (hunk 1 above)
Comment (czgdp1807) [reply sympy/parsing/latex/_antlr/latexparser.py:22]: I hope this isn't a blocker for this PR.

Comment (eric-wieser) [reply sympy/parsing/latex/_antlr/latexparser.py:22]: I don't think #11980 is particularly controversial. What might be a blocker to all these PRs is @asmeurer's opinion in https://github.com/sympy/sympy/pull/19980#issuecomment-675727311

**Comment by asmeurer** [comment]
Can you add tests for this?

What if someone uses Pi or Sigma as a normal variable name? Does this break that, or will it still allow it as long as it doesn't look like a product or sum?

**Comment by eric-wieser** [comment]
> What if someone uses Pi or Sigma as a normal variable name?

That's a really good question, the other one being "why would anyone use them as the operators?"

**Comment by asmeurer** [comment]
I've seen people do it. I think it's mostly just that they don't know about \sum or \prod.

**Comment by oscarbenjamin** [comment]
Do we want this change?

Similar changes were discussed and then removed from #19982.

**Comment by czgdp1807** [comment]
@asmeurer Is this still needed?

**Pull request closed without merging**